src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -500,7 +505,7 @@ export default class GeFormRenderer extends LightningElement{
         this.dispatchEvent(new CustomEvent('togglemodal', { detail }));
     }
 
-    handleSaveBatchGiftEntry(dataImportRecord, formControls) {
+    handleSaveBatchGiftEntry(dataImportRecord, formControls, isCreditCardAuth) {
```

Comment:
I think we can drop `isCreditCardAuth` as a parameter to this function if we opt to look at the payment fields on the Data Import record instead. Would `Payment_Elevate_Batch_Id__c` work for this?